pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy

```diff
@@ -147,105 +137,22 @@ class BuildWheelsTask extends DefaultTask implements SupportsWheelCache, Support
         WheelExtension wheelExtension = ExtensionUtils.getPythonComponentExtension(project, WheelExtension)
         def pythonExtension = ExtensionUtils.getPythonExtension(project)
 
+        def baseEnvironment = environmentMerger.mergeEnvironments([pythonExtension.pythonEnvironment, environment])
+        def wheelAction = new PipWheelAction(packageSettings, project, externalExec, baseEnvironment,
+            pythonDetails, wheelCache, environmentMerger, wheelExtension)
+
         def taskTimer = new TaskTimer()
 
         int counter = 0
         def numberOfInstallables = installables.size()
         installables.each { File installable ->
-            def pyVersion = pythonDetails.getPythonVersion().pythonMajorMinor
             def packageInfo = PackageInfo.fromPath(installable)
             def shortHand = packageInfo.version ? "${ packageInfo.name }-${ packageInfo.version }" : packageInfo.name
```

Comment:
The `PipInstallTask` uses the new `PackageInfo.toShortHand()` method for this.
Can we reuse the same here instead of recalculating?

Reply:
Done